src/components/SecondaryNavDisclosure.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Disclosure
```

Comment:
@MeganSteenkamp Suggestion: There's a fair bit of overlap between PrimaryNavDisclosureButton and this component. Suggest looking at using a single component or a base one